doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -40,7 +40,7 @@ to :ref:`using-a-more-full-featured-chapel` below.
 
          cd chapel-2.0.0
 
-   c. Make sure you are doing the install on a machine with > 4GB of memory.
+   c. Make sure you are doing the install on a machine with > 2GB of memory.
```

Comment:
IMO we could just remove this step from here now, since step 0 is to check the prereqs and the prereqs describe memory requirements in more detail